roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -198,14 +198,27 @@ int ControlUtils::rotateDirection(double currentAngle, double targetAngle){
 }
 
 /// Limits velocity to maximum velocity
-Vector2 ControlUtils::VelocityLimiter(Vector2 vel) {
-    if (vel.length() > rtt::ai::constants::MAX_VEL) {
-        vel = vel.stretchToLength(rtt::ai::constants::MAX_VEL);
+Vector2 ControlUtils::VelocityLimiter(Vector2 vel,double maxVel) {
+    if (vel.length() > maxVel) {
+        vel = vel.stretchToLength(maxVel);
         return vel;
     }
     else return vel;
 }
 
+Vector2 ControlUtils::VelocityLimiter(Vector2 vel, double maxVel,double minVel){
```

Comment:
You can merge this with the previous function; by initializing the parameters in de .h file

Reply:
No, they do functionally different things as one checks for the minimum speed and the other does not.

Reply:
No, since you are then still checking for a minimum speed where in the other one you're not. This gives weird behaviour if minVel>maxVel